package.json

```diff
@@ -12,7 +12,7 @@
     "configs/*"
   ],
   "peerDependencies": {
-    "eslint": "5.12.0",
+    "eslint": "^6.0.1",
```

Comment:
There is still an issue with this version of ESLint. The way we designed the test config to override the rules of the main `.eslintrc` does not work anymore. Maybe something to do with this, I'm not sure: https://eslint.org/docs/user-guide/migrating-to-6.0.0#overrides-precedence.

Quickfix was this in metrics: https://github.com/gynzy/unicorn-metrics/blob/master/test/.eslintrc.js